### PR TITLE
App store 1.9.0

### LINKED ISSUE #41: Big Sur and Apple Silicon Status
TLDR: The App Store version of **SimpleComic won't work in Big Sur on launch day** (today) but should be updated for compatibility soon. Apple Silicon support TBD but expected not to be a problem.

---------

I've been in a multi-step move over the past few months, so haven't had time for side projects but that's closing up now.

For the Mac App store version, I'll look into putting together the newest builds that support Big Sur and Apple Silicon soon.

Looks like Big Sur status was fixed up here: https://github.com/MaddTheSane/Simple-Comic/issues/37

Not sure about Apple Silicon, hopefully it's an easy cross-compile target. I'm getting one of the new MacBooks next week and will test it out then.

### PR DESCRIPTION
**Pull request by nickv2002**
Fixes for https://github.com/MaddTheSane/Simple-Comic/issues/41
Integrates https://github.com/MaddTheSane/Simple-Comic/pull/25

### CODE REVIEW

File: Resources/Base.lproj/TSSTSessionWindow.xib [hunk 1]
```diff
@@ -111,6 +111,27 @@
...
+                        <nil key="toolTip"/>
+                        <size key="minSize" width="51" height="25"/>
+                        <size key="maxSize" width="59" height="27"/>
+                        <segmentedControl key="view" verticalHuggingPriority="750" id="U2R-Pp-TY8">
+                            <rect key="frame" x="0.0" y="14" width="59" height="27"/>
+                            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                            <segmentedCell key="cell" borderStyle="border" alignment="left" style="separated" trackingMode="momentary" id="msg-5l-0tY">
+                                <font key="font" metaFont="system"/>
+                                <segments>
+                                    <segment toolTip="Left Page End" image="NSTouchBarSkipToStartTemplate" width="24" tag="701">
```
Comment (MaddTheSane) [inline Resources/Base.lproj/TSSTSessionWindow.xib]: Probably needs a better tooltip than "Left Page End". Maybe "Left End Page"?

File: Resources/Base.lproj/TSSTSessionWindow.xib [hunk 2]
```diff
@@ -111,6 +111,27 @@
...
+                                        <nil key="label"/>
+                                    </segment>
+                                    <segment toolTip="Right Page Turn" image="NSTouchBarSkipToEndTemplate" width="24" tag="702"/>
```
Comment (MaddTheSane) [inline Resources/Base.lproj/TSSTSessionWindow.xib]: This tooltip definitely needs to be something else.

File: Resources/Base.lproj/TSSTSessionWindow.xib (hunk 1 above)
Comment (nickv2002) [reply Resources/Base.lproj/TSSTSessionWindow.xib]: Set to "Left Page End"

File: Resources/Base.lproj/TSSTSessionWindow.xib (hunk 2 above)
Comment (nickv2002) [reply Resources/Base.lproj/TSSTSessionWindow.xib]: Set to "Right Page End"

**Pull request merged**